### DISCUSSION TITLE
web ui from another machine

### DISCUSSION THREAD

**Discussion by ccbadd** [turboderp/exllama#246] [Q&A]
I run LLMs via a server and I am testing exllama running Ubuntu 22.04 on a Dual Xeon server with 2 AMD MI100s.  Installing exllama was very simple and works great from the console but I'd like to use it from my desktop PC.  Is there an option like Ooobabooga's "--listen" to allow it to be accessed over the local network?

thanks

**Comment by jmoney7823956789378** [comment]
Try this:

https://github.com/turboderp/exllama/blob/82369c52a245278bda49df5629faaef331693092/webui/app.py#L130

Also, what kind of performance are you getting on the MI100s?

**Comment by turboderp** [comment]
Yep, it defaults to hosting the server on `localhost` which means it won't be accessible from other computers. Changing the IP to 0.0.0.0 should fix that. And make sure the port is exposed in the firewall.
